core/maven-example/pom.xml

```diff
@@ -61,28 +62,20 @@
       </plugin>
 
       <plugin>
-        <groupId>org.mortbay.jetty</groupId>
-        <artifactId>maven-jetty-plugin</artifactId>
-
+        <groupId>org.eclipse.jetty</groupId>
+        <artifactId>jetty-maven-plugin</artifactId>
+        <version>9.4.7.v20170914</version>
```

Comment:
CC @olamy just in case he is aware about any issues with that

Reply:
nope

Reply:
FWIW we are using `9.4.5.v20170502` in various other places in the Jenkins project.